### ISSUE TITLE
Error on network_type after migration to 7.5.0

### ISSUE THREAD

**Issue by gonrial** [terraform-aws-modules/terraform-aws-rds-aurora#323]
## Description

After migration to 7.5.0 I got this error:  `An argument named "network_type" is not expected here.`

According to the [hashicorp/aws Changelog](https://raw.githubusercontent.com/hashicorp/terraform-provider-aws/main/CHANGELOG.md) it seems that the `network_type` was introduced by [this ticket](https://github.com/hashicorp/terraform-provider-aws/issues/26489) and released in the `4.30.0` release, so it's not supported in the `4.28.0` which is the minimum version of this module.

## Versions

- Module version [Required]: 7.5.0

- Terraform version: Terraform v1.1.8 on linux_amd64

- Provider version(s): 
  + provider registry.terraform.io/hashicorp/aws v4.29.0
  + provider registry.terraform.io/hashicorp/random v3.4.1

## Reproduction Code [Required]

```
tf plan
```

### Terminal Output Screenshot(s)

```
│ Error: Unsupported argument
│ 
│   on .terraform/modules/db/main.tf line 92, in resource "aws_rds_cluster" "this":
│   92:   network_type                  = var.network_type
│ 
│ An argument named "network_type" is not expected here.
```

**Comment by bryantbiggs** [comment]
my apologies - looking again at the changelog it looks like the RDS instance resources was updated for this and the RDS cluster was not added till 4.30.0 as you've stated.

**Comment by antonbabenko** [comment]
This issue has been resolved in version 7.5.1 :tada:
